app/utils/process-contribution-data.js

```diff
@@ -0,0 +1,21 @@
+export default function processContributionData(data) {
+  const processed = {}
+
+  if (data.confirmedAtBlock && (typeof data.confirmedAtBlock.toNumber === 'function')) {
+    processed.confirmedAt = data.confirmedAtBlock.toNumber();
```

Comment:
was wondering if the rename from `confirmedAtBlock` to `confirmedAt` is so needed because now we call it differently in the kredits-contracts and kredits-web

Reply:
It's not a rename, because it's the same property that returned the number before as well. The `confirmedAtBlock` is the one from the contracts, which isn't formatted as a number.